### ISSUE TITLE
ERROR:  Invalid character (�) in string

### ISSUE THREAD

**Issue by e02d96ec16** [forcedotcom/cli#35]
### Summary

Failure to retrieve valid metadata, if it contains characters deemed invalid by the XML parser.

### Steps To Reproduce:

1.  Create a community
    1. Name: 😂
2.  `sfdx force:source:pull -f`

### Expected result

Retrieved the metadata for this in /sites

### Actual result

Retrieval fails with:
> ERROR:  Invalid character (�) in string: 😂 Profile at index 0.

### Additional information

```
Error: Invalid character (�) in string: 😂 Profile at index 0
    at XMLStringifier.module.exports.XMLStringifier.assertLegalChar («···»\node_modules\sfdx-cli\node_modules\salesforce-alm\node_modules\xmlbuilder\lib\XMLStringifier.js:149:15)
⁞
```

This is related to https://github.com/oozcitak/xmlbuilder-js/issues/154 which got adopted in https://github.com/Leonidas-from-XIV/node-xml2js/pull/395 therefore salesforce-alm would just need to bump its xml2js dependency (please).

**SFDX CLI Version**:
`sfdx-cli/6.53.0 (windows-x64) node-v11.10.0`

**SFDX plugin Version**:
`salesforcedx 45.2.0 (core)`

**OS and version**:
```
Caption        : Microsoft Windows 10 Pro
OSArchitecture : 64-bit
Version        : 10.0.17134
UBR            : 590
ReleaseId      : 1803
```

**Comment by dcarroll** [comment]
You are not seriously trying to name a community with an emoji are you? I don't think you are, so, if you can represent the name differently that would help.

**Comment by clairebianchi** [comment]
@e02d96ec16 are you looking for a better error message?

**Comment by e02d96ec16** [comment]
The point is sfdx does not support something that Salesforce does (ie emojis (possibly other unicode too?) in Community Name).

I appreciate this is an edge case, in which case if it's not to be supported (now), I would hope for it to at least be officially documented as a Known Issue.

Also, I apologise if this sounded like I was seeking for help. I'm not (I already fixed it locally by applying what I explained in Additional Information); I was just trying to share my findings, hoping to improve sfdx.
